modules/nf-core/rseqc/inferexperiment/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process RSEQC_INFEREXPERIMENT"
+    script "../main.nf"
+    process "RSEQC_INFEREXPERIMENT"
+    config "./nextflow.config"
+    
```

Comment:
```suggestion
```

Reply:
Can I keep the spaces? I actually find it much easier to read with things spaced out a bit

Reply:
I see no issues with keeping an empty line to enhance readability